From b702985deea7b7638b48ffb53e39933336d4cd0c Mon Sep 17 00:00:00 2001
From: Thor Thayer <thor.thayer@linux.intel.com>
Date: Wed, 30 Oct 2019 10:30:40 -0500
Subject: [PATCH 006/120] HSD #1707095796: Add Agilex SMMU clocks

commit 6c45822f248968bab93a36c4c52a5835d328dd35 from
https://github.com/altera-opensource/linux-socfpga.git

arm64: dts: agilex: add SMMU clocks

Add the SMMU clocks needed for Agilex SMMU.

Has dependency on HSD #2209582664:
https://pg-rdcrucible.altera.com:8080/cru/LINUX-SOCFPGA-1535

Signed-off-by: Thor Thayer <thor.thayer@linux.intel.com>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 arch/arm64/boot/dts/intel/socfpga_agilex.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi b/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
index 9d1d8c1f58c0..bd5cc1dc28dd 100644
--- a/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
+++ b/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
@@ -350,6 +350,9 @@
 				<0 162 4>, <0 163 4>, <0 164 4>, <0 165 4>,
 				<0 166 4>, <0 167 4>, <0 168 4>, <0 169 4>;
 			stream-match-mask = <0x7ff0>;
+			clocks = <&clkmgr AGILEX_MPU_CCU_CLK>,
+				 <&clkmgr AGILEX_L3_MAIN_FREE_CLK>,
+				 <&clkmgr AGILEX_L4_MAIN_CLK>;
 			status = "disabled";
 		};
 
-- 
2.31.1

